docs/training.mdx

```diff
@@ -0,0 +1,256 @@
+
+Infrastructure data
+
+| Component | Disclosed data |
+| --------- | -------------- |
+| GPU | Nvidia A100 80GB |
+| Server | HPE Apollo 6500 Gen10 Plus |
+| Number of GPUs | 384 |
+| Number of servers | 48 |
+| Training location | France |
+
+Training data
+
+| Component | Disclosed data |
+| --------- | -------------- |
+| Total reserved time | 118 days |
+| Reservation start time | January 2022 (?) |
+| GPU hours for final model | 1,082,990 |
+
+## Normalization of disclosed data
+
+When disclosed data is not present or not complete, we need to use predictive or heuristic data to fill in the gaps.
+
+| Missing data point | Mechanism to replace |
+| - | - |
+| GPU model | Use the most common GPU for the training year (for instance, 2022 is Nvidia A100) |
+| Server model | Use the most common server or instance type for the training year |
+| GPUs used | Use the average cluster size for similar models |
+| Servers used | Divide GPUs used by average GPUs per server |
+| Location | Use the US as a relatively high-carbon country |
+| Datacenter PUE | Use location average |
+| Datacenter WUE | Use location average |
+| Total reserved time | Use the average ratio of reserved time to GPU hours |
+| Reservation start time | Use the published model date minus the total reserved time |
+| GPU hours for final model | Predict using parameters and architecture per [OpenCarbonEval](https://arxiv.org/pdf/2405.12843) |
+| GPU hours for intermediate models | Predict based on ratio of final to intermediate for other disclosed models |
+
+### Example normalization: BLOOM 176B
+
+The BLOOM paper includes most of the required parameters for carbon emissions. However, it does not include data for water consumption. We use the fallbacks for the missing data points in the calculations below.
+
+Some assumptions:
+- The 35.8 tons of CO2 for the intermediate models implies proportional reservation and usage as the final model (1.45x)
+
+Normalized training data
+
+| Component | Disclosed data |
+| --------- | -------------- |
+| Total reserved time | 289 days (118 x 1.45) |
+| Reservation start time | August 2021 (finish date of June 2022 minus 289 days) |
+| GPU hours for final model | 2,653,326 (1,082,990 x 1.45) |
+
+## Calculation of carbon emissions
+
+To calculate CO2 emissions, we use the [Software Carbon Intensity formula](https://localhost:3000/overview#co2-intensity). We need a few data points:
+
+- The [usage energy per GPU hour](/cluster#energy-perone-gpu-hour) for the cluster
+- The [embodied emissions per reserved hour](/cluster#embodied-emissions) for the cluster
+- The carbon intensity of the grid in the training location(s) during the training time period
+
+The training emissions will be:
+
+```
+Total emissions = (embodied emissions) + (usage emissions)
+
+Embodied emissions = (cluster embodied emissions per hour) x (training time)
+
+Usage emissions = (usage energy per GPU-hour) x (total GPU hours) x (average grid intensity during training)
```

Comment:
if instead of total GPU hours, we had GPU hours utilized by hour/time, we could potentially leverage the actual grid mix during that hour. i'm thinking of cases where potentially training could be paused and resumed to only happen during times where marginal grid mix intensity is low

Reply:
Yep agreed - though I was thinking this would be easiest to represent as a lower average grid mix?